manim/mobject/three_dimensions.py

```diff
@@ -122,6 +134,152 @@ def func(self, u, v):
         return np.array([np.cos(v) * np.sin(u), np.sin(v) * np.sin(u), np.cos(u)])
 
 
+class Cone(ParametricSurface):
+    CONFIG = {
+        "base_radius": 1,
+        "height": 1,
+        "direction": Z_AXIS,
+        "show_base": False,
+        "resolution": 24,
+        # v will stand for phi
+        "v_min": 0,
+        "v_max": TAU,
+        # u will stand for r
+        "u_min": 0,
+        # u_max is calculated as a property
+        "checkerboard_colors": False,
+    }
+    """
+               |\\
+               |_\\ <-- theta
+    height --> |  \\
+               |   \\ <-- r
+               |    \\
+               |     \\
+               --------
+               base_radius
+    """
```

Comment:
I believe this might be the reason the docs aren't building. Can you please add a more standard docstring to this class instead?

Reply:
I changed it to a more standard one (though I liked this triangle :P)

Reply:
Moving the triangle inside a `.. code-block::` also would have worked, I suppose.